action_server_bt/src/action_server_bt.cpp

```diff
@@ -116,22 +122,28 @@ void ActionServerBT::execute(const std::shared_ptr<GoalHandleActionTree> goal_ha
       {
         action_result->error_message = "Action Server canceling and halting Behavior Tree";
```

Comment:
Note to self that I'm missed setting the result->node_status when canceling.
`action_result->node_status = convert_node_status(status);`

Reply:
fixed